repobuilder/job.go

```diff
@@ -281,9 +283,10 @@ func (j *Job) Run(ctx context.Context) {
 		j.AddError(errors.Wrapf(err, "problem getting s3 bucket %s", j.Distro.Bucket))
 		return
 	}
-
 	defer j.MarkComplete()
 
+	bucket = pail.NewParallelSyncBucket(pail.ParallelBucketOptions{Workers: runtime.NumCPU() * 2}, bucket)
```

Comment:
This should just be `runtime.NumCPU()` (it is already the logical number of CPUs)

Reply:
Given that there isn't anything else going on on the system, I think some multiplicative factor might be useful.